pkg/apis/hive/v1/openstack/platform.go

```diff
@@ -13,4 +13,7 @@ type Platform struct {
 	// Cloud will be used to indicate the OS_CLOUD value to use the right section
 	// from the cloud.yaml in the CredentialsSecretRef.
 	Cloud string `json:"cloud"`
+
+	// TrunkSupport indicates whether or not to use trunk ports in your OpenShift cluster.
+	TrunkSupport bool `json:"trunkSupport"`
```

Comment:
Is this a required field? Or does it default to false?

Reply:
I believe it defaults to false.

Reply:
If there is a default, then this should be `omitempty` and `optional`.